tests/e2e/tests/content-manager/relations.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from '@playwright/test';
+import { login } from '../../utils/login';
+import { resetDatabaseAndImportDataFromPath } from '../../utils/dts-import';
+import { describeOnCondition } from '../../utils/shared';
+
+describeOnCondition(process.env.STRAPI_FEATURES_UNSTABLE_RELATIONS_ON_THE_FLY === 'true')(
+  'Relations on the fly tests',
```

Comment:
I would prefix the unstable so it's clear the test and feature are unstable

```suggestion
  'Unstable Relations on the fly',
```